### ISSUE TITLE
Change term - vernacularName

### ISSUE THREAD

**Issue by nielsklazenga** [tdwg/dwc#524]
## Term change

* Submitter: Niels Klazenga
* Efficacy Justification (why is this change necessary?): The current definition is self-referencing
* Demand Justification (if the change is semantic in nature, name at least two organizations that independently need this term): 
    The proposed definition is the definition that is accepted for the IRI equivalent in TCS. The change is semantic in nature only to the extent that it changes the wording of the definition; the meaning of the term remains unchanged.
* Stability Justification (what concerns are there that this might affect existing implementations?): None
* Implications for dwciri: namespace (does this change affect a dwciri term version)?: 
    IRI equivalents for terms that are organised in the Taxon class are not in the `dwciri` namespace, but the `tcs` namespace. It is proposed that the definition of the TCS term is adopted,

Current Term definition: https://dwc.tdwg.org/list/#dwc_vernacularName

Proposed attributes of the new term version (Please put actual changes to be implemented in **bold** and ~strikethrough~):

* Term name (in lowerCamelCase for properties, UpperCamelCase for classes): vernacularName
* Term label (English, not normative): Vernacular Name
* Organized in Class (e.g., Occurrence, Event, Location, Taxon): Taxon
* Definition of the term (normative): ~A common or vernacular name~**Name for a taxon in a language used for general purposes**.
* Usage comments (recommendations regarding content, etc., not normative): 
* Examples (not normative): 
    `Andean Condor`
    `Condor Andino`
    `American Eagle`
    `Gänsegeier`
* Refines (identifier of the broader term this term refines; normative): N/A
* Replaces (identifier of the existing term that would be deprecated and replaced by this term; normative): N/A
* ABCD 2.06 (XPATH of the equivalent term in ABCD or EFG; not normative): N/A

**Comment by qgroom** [comment]
Your proposed definition is also self-referencing.
Also, vernacular names are not necessarily used for general purposes.

`A term used in a language to refer to a taxon, often varying by region or community and distinct from its scientific designation.`

**Comment by nielsklazenga** [comment]
@qgroom, it is not and it is the language that is used for general purposes. It is also not really my definition, as I took the first part of the definition in [Hawksworth 2010](https://www.gbif.org/document/80577/terms-used-in-bionomenclature-the-naming-of-organisms-and-plant-communities), which goes a bit off the rails after that.

**Comment by tucotuco** [comment]
Is "organism" the right target for the term? dwc:vernacularName is currently organized in the dwc:Taxon class and so these names are alternative names for something taxonomic. I would like to avoid confusion of a dwc:Taxon term also being a dwc:Organism term. It is clearly not the latter.

**Comment by nielsklazenga** [comment]
Yes, that is an issue for me too. The 'organism' is not such a problem in TCS, but it still does not sound right (actually, as TCS is meant to complement Darwin Core, it is just as big an issue in TCS). I am happy to replace 'organism or organisms' with 'taxon'. In Darwin Core you could use 'dwc:Taxon', which would be a bit of an issue for TCS, as we borrow the Darwin Core term in TCS where it is used on a tcs:TaxonConcept (the tcs:vernacularName is an IRI property ).

**Comment by tucotuco** [comment]
I would appreciate it if the interested parties could submit a complete change request with all of the attributes in the [Term change template](https://github.com/tdwg/dwc/issues/templates/edit).

**Comment by nielsklazenga** [comment]
@tucotuco , will do.

**Comment by tucotuco** [comment]
@nielsklazenga Can you follow up on this so we can get this into the Darwin Core Data Package review? You'd just need to edit the first comment, prefacing what is there with a completed Term - change template.

**Comment by nielsklazenga** [comment]
You cannot define a term by listing a couple of attributes and saying what it is not. The only part in the definition above that ualifies as a definition is the bit before the first comma and that can define any type of name. Just about every Darwin Core term is in a language and `dwc:scientificName` also refers to a taxon.

So, to explain the definition we chose for TCS:

1. We first need to indicate that a vernacular name is a name for a taxon, which is where the '**Name for a taxon**' bit comes from
2. Then we need to distinguish vernacular names from other taxon names, _e.g._ `dwc:scientificName`, which is where the '**in a language used for general purposes**' comes in. We could also have said something like '**in a language or dialect that is spoken by people in a particular country or region**', but for TCS we went with something more similar to the existing definition in [Hawksworth 2010](https://www.gbif.org/document/80577/terms-used-in-bionomenclature-the-naming-of-organisms-and-plant-communities).

I think it will be good if `dwc:vernacularName` and `tcs:vernacularName` have the same definition, as they are each other's Literal and IRI counterparts, so I propose to table this proposal until TCS has been ratified (it is before the TDWG Executive at the moment, so we cannot make any changes at this time), so we can change the definitions in both standards at the same time. In the meantime there is a proposal (#725) to change the examples of `dwc:vernacularName`, which should be dealt with first.
